db/structure.sql

```diff
@@ -915,224 +965,245 @@ ALTER TABLE ONLY users
+-- Name: index_study_impacts_on_study_id; Type: INDEX; Schema: public; Owner: -; Tablespace: 
 --
 
 CREATE INDEX index_study_impacts_on_study_id ON study_impacts USING btree (study_id);
 
 
 --
--- Name: index_study_notes_on_study_id; Type: INDEX; Schema: public; Owner: -; Tablespace:
+-- Name: index_study_notes_on_study_id; Type: INDEX; Schema: public; Owner: -; Tablespace: 
 --
 
 CREATE INDEX index_study_notes_on_study_id ON study_notes USING btree (study_id);
 
 
 --
--- Name: index_study_settings_on_name; Type: INDEX; Schema: public; Owner: -; Tablespace:
+-- Name: index_study_settings_on_name; Type: INDEX; Schema: public; Owner: -; Tablespace: 
 --
 
 CREATE UNIQUE INDEX index_study_settings_on_name ON study_settings USING btree (name);
 
 
 --
--- Name: index_study_stages_on_name; Type: INDEX; Schema: public; Owner: -; Tablespace:
+-- Name: index_study_stages_on_name; Type: INDEX; Schema: public; Owner: -; Tablespace: 
 --
 
 CREATE UNIQUE INDEX index_study_stages_on_name ON study_stages USING btree (name);
 
 
 --
--- Name: index_study_topics_on_name; Type: INDEX; Schema: public; Owner: -; Tablespace:
+-- Name: index_study_topics_on_name; Type: INDEX; Schema: public; Owner: -; Tablespace: 
 --
 
 CREATE UNIQUE INDEX index_study_topics_on_name ON study_topics USING btree (name);
 
 
 --
--- Name: index_study_types_on_name; Type: INDEX; Schema: public; Owner: -; Tablespace:
+-- Name: index_study_types_on_name; Type: INDEX; Schema: public; Owner: -; Tablespace: 
 --
 
 CREATE UNIQUE INDEX index_study_types_on_name ON study_types USING btree (name);
 
 
 --
--- Name: index_users_on_confirmation_token; Type: INDEX; Schema: public; Owner: -; Tablespace:
+-- Name: index_users_on_confirmation_token; Type: INDEX; Schema: public; Owner: -; Tablespace: 
 --
 
 CREATE UNIQUE INDEX index_users_on_confirmation_token ON users USING btree (confirmation_token);
 
 
 --
--- Name: index_users_on_email; Type: INDEX; Schema: public; Owner: -; Tablespace:
+-- Name: index_users_on_email; Type: INDEX; Schema: public; Owner: -; Tablespace: 
 --
 
 CREATE UNIQUE INDEX index_users_on_email ON users USING btree (email);
 
 
 --
--- Name: index_users_on_msf_location_id; Type: INDEX; Schema: public; Owner: -; Tablespace:
+-- Name: index_users_on_msf_location_id; Type: INDEX; Schema: public; Owner: -; Tablespace: 
 --
 
 CREATE INDEX index_users_on_msf_location_id ON users USING btree (msf_location_id);
 
 
 --
--- Name: index_users_on_reset_password_token; Type: INDEX; Schema: public; Owner: -; Tablespace:
+-- Name: index_users_on_reset_password_token; Type: INDEX; Schema: public; Owner: -; Tablespace: 
 --
 
 CREATE UNIQUE INDEX index_users_on_reset_password_token ON users USING btree (reset_password_token);
 
 
 --
--- Name: unique_schema_migrations; Type: INDEX; Schema: public; Owner: -; Tablespace:
+-- Name: unique_schema_migrations; Type: INDEX; Schema: public; Owner: -; Tablespace: 
```

Comment:
Hmm, weird - but yeah, that file will get replaced whenever we next do a db migration (which will be very soon), so it's probably not worth worrying too much about at the moment.